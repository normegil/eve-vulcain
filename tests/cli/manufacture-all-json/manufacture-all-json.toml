bin.name = "eve-vulcain"
args = "manufacture all --json -vvv"
status.code = 0
