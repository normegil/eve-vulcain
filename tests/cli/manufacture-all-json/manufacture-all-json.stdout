{"items":[{"name":"Drone Link Augmentor I","regional_average_volume":1331,"profits_per_hour":-836802.7909106829},{"name":"Damage Control II","regional_average_volume":10822,"profits_per_hour":-944809.6717957256}]}
